### DISCUSSION TITLE
如何调整检测字符间距呢，或者有参数配置吗

### DISCUSSION THREAD

**Discussion by kwdchol** [PaddlePaddle/PaddleOCR#12335] [Q&A] [answered]
图片里 两个字符间距不是很宽 但是也会检测成两个框 
有没有那个参数可以配置  如果大于某个间距才会检测为两个分开的字符 有两个框

**Comment by bucaiLi** [comment]
好兄弟 你解决了嘛 我也先知道怎么才能检测出每个汉字的坐标而不是一段话的坐标   这个是有参数的嘛

**Reply by bruceking0825** [reply]
现在rapidocr可以做字符级检测，但字符的坐标总是差那么一点，不知道要不要重新训练字符集，正在研究

**Comment by kwdchol** [comment]
> 好兄弟 你解决了嘛 我也先知道怎么才能检测出每个汉字的坐标而不是一段话的坐标 这个是有参数的嘛

没有 不过我的需求和你不一样  我是想知道如何调整检测间距

**Comment by tink2123** [comment, marked as answer]
推理的时候可以打开 --dilation=True 这个参数，膨胀分割图可能会缓解。如果还是会分开检测可以尝试自己重新标注数据训练一下，

**Comment by kwdchol** [comment]
> 推理的时候可以打开 --dilation=True 这个参数，膨胀分割图可能会缓解。如果还是会分开检测可以尝试自己重新标注数据训练一下，

好的 感谢 我试试
